src/components/PromotionList.tsx

```diff
@@ -166,7 +166,7 @@ export default function PromotionList(props: Props): ReactElement {
     PromotionService.queryPromotions(
       promotionsState.filter,
       promotionsState.sort,
-      authUser ? authUser.user.id : undefined
+      authUser?.user?.id
```

Comment:
Ah, I didn't know you could do this. This looks much cleaner

Reply:
hehe yeah magic of elvis operators, I think that's what theyre called